packages/utils/src/eslint-utils/rule-tester/RuleTester.ts

```diff
@@ -63,6 +63,9 @@ function isDescribeWithSkip(
   );
 }
 
+/**
+ * @deprecated use `@typescript-eslint/rule-tester` instead
```

Comment:
Can you also file a delection tracking issue & link to it here? I'm guessing it'll be a v7 breaking change.